kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/event/PlayerEventHolder.kt

```diff
@@ -36,8 +36,16 @@ class PlayerEventHolder {
     private var _onPlaybackMetadata = MutableSharedFlow<PlaybackMetadata>(1)
     var onPlaybackMetadata = _onPlaybackMetadata.asSharedFlow()
 
-    private var _onMediaSessionCallbackTriggered = MutableSharedFlow<MediaSessionCallback>()
-    var onMediaSessionCallbackTriggered = _onMediaSessionCallbackTriggered.asSharedFlow()
+    private var _onPlayerActionTriggeredExternally = MutableSharedFlow<MediaSessionCallback>()
+
+    /**
+     * Use these events to track whenever a player action has been triggered from an outside source.
+     *
+     * The sources can be: media buttons on headphones, Android Wear, Android Auto, Google Assistant, media notification, etc.
+     *
+     * For this observable to send events, set [interceptPlayerActionsTriggeredExternally][com.doublesymmetry.kotlinaudio.models.PlayerConfig.interceptPlayerActionsTriggeredExternally] to true.
+     */
+    var onPlayerActionTriggeredExternally = _onPlayerActionTriggeredExternally.asSharedFlow()
```

Comment:
You can now opt in to have the player actions be intercepted. For RNTP this will be `true` by default, but since KA is used as a general audio library, it's good to have the option.